### ISSUE TITLE
[Experiment] Reduce regularization and try a lower LR

### ISSUE THREAD

**Issue by AdityaKane2001** [AdityaKane2001/regnets_trainer#5]
As in #4, the train-val variance is very small compared to the current-goal bias. Thus, I want to run a HP sweep for weight decay and learning rate. Another way to reduce regularization is to tame the inception crop. As you said earlier, increasing the area factor will cover that.

**Comment by sayakpaul** [comment]
So, are you distributing a single experiment over many workers spread across many machines? Seems like overkill to me, if so.
